esmvaltool/cmorizers/obs/download_scripts/download_merra2.sh

```diff
@@ -10,14 +10,30 @@
 #  - Check the years
 #  - Run this script as follow ./download_merra2.sh path_to_save_the_files
 #
-# 
+#
 # Additional resources:
 #    - file specification document is available here: https://gmao.gsfc.nasa.gov/pubs/docs/Bosilovich785.pdf)
 
+if [[ $# -eq 0 ]] ; then
+    echo 'Please provide the download directory as first argument to this script.'
+    exit 1
+fi
 
 # The years that should be downloaded.
-yrs=({1980..2019})
+yrs=({1980..2020})
+
 
+# Monthly data
+# for yr in ${yrs[@]}; do
+#    echo $yr
+#    wget -r -nd -nH --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --directory-prefix=$1 --auth-no-challenge=on --keep-session-cookies -A '*_lnd_*.nc4' https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2_MONTHLY/M2TMNXLND.5.12.4/${yr}/ -R "*.tmp"
+# done
+
+# Daily data
 for yr in ${yrs[@]}; do
-    wget -r -nd -nH --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --directory-prefix=$1 --auth-no-challenge=on --keep-session-cookies -A '*_lnd_*.nc4' https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2_MONTHLY/M2TMNXLND.5.12.4/${yr}/ -R "*.tmp"
+   echo $yr
+   wget -r -nd -nH --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies --directory-prefix=$1 --auth-no-challenge=on --keep-session-cookies -A '*_lnd_*.nc4' https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2/M2T1NXLND.5.12.4/${yr}/ -R "*.tmp"
 done
+
+# Also download constants file
+# wget --load-cookies ~/.urs_cookies --save-cookies ~/.urs_cookies directory-prefix=$1 --auth-no-challenge=on --keep-session-cookies https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2_MONTHLY/M2C0NXLND.5.12.4/1980/MERRA2_100.const_2d_lnd_Nx.00000000.nc4
```

Comment:
@bascrezee I know this is not of your addition and this shell script was here before the changes you are applying, but can we maybe make this either a python script or add the bash syntax as documentation? I don't like having shell scripts embedded in source code, and I think @bouweandela can agree to this too - not entirely safe and not supporting the tcsh or other types of shells. Don't necessarily have to be done in this PR tho

Reply:
@jvegasbsc Is working on the download functionality in #1657, so I'll leave it up to him to decide whether shell scripts are a good idea or not, but indeed Python is generally a safer and more convenient choice.